### DISCUSSION TITLE
Cost of Entity Function

### DISCUSSION THREAD

**Discussion by Planche95** [Azure/azure-functions-durable-extension#1912] [Q&A] [answered]
Where I can find information about Durable Entity Function Cost? Is it Function Cost + Azure Storage transactions cost?
I found this: https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-billing but there is nothing abour Entity functions

**Comment by ConnorMcMahon** [comment, marked as answer]
Entity functions don't have replay behavior, which is why we don't spend any time detailing odd billing behavior for them, as they are very straightforward. It is just the Function cost and the Azure Storage transactions costs.
